doc/programming/parsing.rst

```diff
@@ -551,10 +561,10 @@ The most commonly used hooks are:
 ``on <field name> foreach { ... }`` (container hook)
     Assuming the specified field is a container (e.g., a vector), this
     executes each time a new container element has been parsed, and
-    just before it's been added to the container. The parsed element
-    is accessible through the ``$$`` identifier, and can be modified
-    before it's stored. The hook implementation may also use the
-    :ref:`statement_stop` statement to abort container parsing,
+    just before it's been added to the container. The element's final
+    value is accessible through the ``$$`` identifier, and can be
+    modified before it's stored. The hook implementation may also use
```

Comment:
What do you think about this:

```suggestion
    value is accessible through the ``$$`` identifier, although it can be
    further modified before it's stored. The hook implementation may also use
```